core/src/components/input/input.vars.scss

```diff
@@ -2,3 +2,9 @@
 
 /// @prop - How much to scale the floating label by
 $input-floating-label-scale: 0.75 !default;
+
+/// @prop - The disabled opacity of the input text, the label, helper text, char counter and placeholder of a disabled input.
+/// This value comes from the Material Design spec:
+/// - https://github.com/material-components/material-web/pull/549
+/// We apply the same value to both iOS and MD themes.
+$input-disabled-opacity: 0.37 !default;
```

Comment:
I'm fine with converging on one opacity. The difference between 0.4 and 0.37 is so small that it likely will not have a noticeable impact on apps. However, the only reason I decided to converge on `$input-floating-label-scale` for both iOS and MD is that iOS has no concept of a floating label, so we are free to pick a reasonable default. This is not the case with a disabled text field as this pattern appears on iOS. I'd prefer we only converge on patterns when no pattern exists for a certain platform and we need to create one.